crates/criticalup-cli/src/commands/install.rs

```diff
@@ -269,12 +272,36 @@ mod tests {
 
     // Check if there is a revoked content Sha256 in the package.
     #[test]
-    fn revocation_check() {
+    fn revocation_check_normal() {
         let revocation_info =
             RevocationInfo::new(vec![PACKAGE_SHA256.into()], datetime!(2400-10-10 00:00 UTC));
         let release = generate_release();
         assert!(matches!(
-            check_for_revocation(&revocation_info, &release),
+            check_for_revocation(&revocation_info, &release, false),
+            Err(RevocationCheckFailed(..))
+        ))
+    }
+
+    // Offline mode but valid expiration date.
+    #[test]
+    fn revocation_check_offline() {
+        let revocation_info =
+            RevocationInfo::new(vec![PACKAGE_SHA256.into()], datetime!(2400-10-10 00:00 UTC));
+        let release = generate_release();
+        assert!(matches!(
+            check_for_revocation(&revocation_info, &release, true),
+            Err(RevocationCheckFailed(..))
+        ))
+    }
+
+    // The expired datetime must be ignored in Offline mode.
+    #[test]
+    fn revocation_check_offline_mode_expired_datetime() {
+        let revocation_info =
+            RevocationInfo::new(vec![PACKAGE_SHA256.into()], datetime!(2012-10-10 00:00 UTC));
+        let release = generate_release();
+        assert!(matches!(
+            check_for_revocation(&revocation_info, &release, true),
```

Comment:
Hmm.... Shouldn't this one be `Ok(())`?

Reply:
How do I mess up something so easy is beyond me. You are correct it should be `Ok(())`.

Reply:
ngl I spent a solid few minutes glaring at it wondering if I was just gone silly.